### PR TITLE
[unipool-univ2-lp] Add unipool-univ2-lp strategy

### PR DESCRIPTION
**Pull request by dapplion**
Generic strategy for a Unipool contract where you stake a UniV2 LP token from a pool with the earned token.

e.g.
- Unipool earns UNI
- Unipool stakes UniV2 LP UNI-ETH

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Please add `unipool-univ2-lp` to `strategies/index.ts` object..

File: src/strategies/unipool-univ2-lp/examples.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "symbol": "TOKN",
+        "decimals": 18,
+        "tokenAddress": "0x39A2fad92db0ba608869115eB2E6d26eF16CABC5",
+        "unipoolAddress": "0x2c46430D4dA467Cf247E49A33ECd27DE1e16F451",
+        "lpTokenAddress": "0x415bd1074305ac16d3da4f6f1fcac7dd9d9b61a6"
+      }
+    },
+    "network": "4",
+    "addresses": ["0xcA5720267Fcb7ccA13007AfA096ae01B0864f765"],
+    "snapshot": "8947700"
```
Comment (ChaituVR) [inline src/strategies/unipool-univ2-lp/examples.json:16]:
```suggestion
    "snapshot": 8947700
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/dappnode-dao/examples.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        "unipoolSameTokenXdai": "0xbC396689893D065F41bc2C6EcbeE5e0085233447"
+      }
+    },
+    "network": "1",
+    "addresses": [
+      "0x09212c58107c8da21b0f67a63e2144bb68bee4eb",
+      "0x8506d946CC63d1F1F3a303d68B0Da64597Cd64F3",
+      "0xc5df4d5ed23f645687a867d8f83a41836fcf8811"
+    ],
+    "snapshot": "11635000"
```
Comment (bonustrack) [inline src/strategies/dappnode-dao/examples.json:24]: This should be a number
```suggestion
    "snapshot": 11635000
```

Comment (dapplion) [reply src/strategies/dappnode-dao/examples.json]: Removed un-related second strategy from this PR

**Comment by dapplion** [comment]
Should be ready to review and merge now

**Comment by bonustrack** [comment]
@dapplion Can you remove state.ssz file? Not sure what is this about

**Comment by dapplion** [comment]
> @dapplion Can you remove state.ssz file? Not sure what is this about

Deleted sorry. Casually the state of Eth2.0 mainnet

**Pull request merged**